NitroxServer/Serialization/ServerConfig.cs

```diff
@@ -9,6 +9,8 @@ namespace NitroxServer.Serialization
     public class ServerConfig : NitroxConfig<ServerConfig>
     {
         private int maxConnectionsSetting = 100;
+
+        private int initialSyncTimeoutSetting = 120000;
```

Comment:
I would double this value if not triple. We know we have a few players with shitty internet/PCs and there will be a bunch of them in the help channel "WHY NOT WORKING?????"